README.md

```diff
@@ -36,7 +36,6 @@ Git user attention
 
 3. After running `download-deps.py`.
 
-         $ cd cocos2d-x
```

Comment:
Oh, I see, there is another `cd cocos2d-x` in the previous instruction.

I think it's ok to delete this line, but can you append `cocos2d-x ` in the front of `$`, that would be helpful, thank you